tests/Functional/FunctionalTestCase.php

```diff
@@ -28,7 +30,8 @@ abstract class FunctionalTestCase extends TestCase
      */
     protected function setUpContainer($testCase, $commonFile = 'common.yml')
     {
-        $this->kernel = new TestKernel($testCase, $commonFile);
+        $prefix = Kernel::MAJOR_VERSION <= 4 ? 'legacy_' : '';
```

Comment:
The reason why tests for Symfony 5.x are failing is because we need to set `security.firewalls.main.anonymous` to `true` for Symfony 5. So, we might need a separate configuration file just for Symfony 5.

Reply:
As far as I can see they [deprecated](https://github.com/symfony/symfony/blob/v5.1.0/UPGRADE-5.1.md#securitybundle) `anonymous: lazy` in favor of `lazy: true`.